AEPCore/Sources/eventhub/EventHub.swift

```diff
@@ -48,7 +48,7 @@ final class EventHub {
         // Setup eventQueue handler for the main OperationOrderer
         eventQueue.setHandler { [weak self] (event) -> Bool in
             guard let processedEvent = self?.preprocessors.shallowCopy.reduce(event, {$1($0)}) else { return true }
-
+            Log.debug(label: self?.LOG_TAG ?? "EventHub", "Processed event: \(processedEvent)")
```

Comment:
Can you change this to trace and also add additional check as it lies in hot path?
if Log.logFilter == .trace {
  Log.trace(label: self?.LOG_TAG ?? "EventHub",
                      "Processed Event #\(String(describing: self?.eventNumberMap[event.id] ?? 0)) - \(processedEvent)")
}

Reply:
Can you also add a similar check in dispatch(event: Event) function

Reply:
Why do we need the additional check? The Log.trace function does the check in the function.